helm/kfp-operator/templates/eventing/event-source-webhook.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.statusFeedback.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: {{ include "kfp-operator.fullname" . }}-event-source-webhook
```

Comment:
So this generates a pod name of `kfp-operator-event-source-webhook-eventsource-rx2t4-7f4bf6bv8lv` 🙈

Reply:
{{ include "kfp-operator.fullname" . }}-events-webhook maybe? 🤔

Reply:
don't see much point in the small difference in pod name given it [will be removed in step 2](https://github.com/sky-uk/kfp-operator/issues/357)

Reply:
I believe eventsource is always without the hypen and argo adds it for you so the name should just be what is specific to it

Reply:
Addressed in 86457a9731f9a306c4d88c4091ebac780efe759d